lib/awesome_nested_set/awesome_nested_set.rb

```diff
@@ -98,9 +98,9 @@ def acts_as_nested_set_relate_parent!
           :primary_key => primary_column_name,
           :counter_cache => acts_as_nested_set_options[:counter_cache],
           :inverse_of => (:children unless acts_as_nested_set_options[:polymorphic]),
-          :polymorphic => acts_as_nested_set_options[:polymorphic],
           :touch => acts_as_nested_set_options[:touch]
         }
+        options[:polymorphic] = true if acts_as_nested_set_options[:polymorphic]
```

Comment:
```
options[:polymorphic] = acts_as_nested_set_options[:polymorphic] if acts_as_nested_set_options[:polymorphic]
```

Reply:
that's hilarious, I was going to make the *exact* same suggestion

Reply:
@filippoliverani what do you think of the above? ^^